src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SemanticTokensEditsDiffer.cs

```diff
@@ -3,48 +3,45 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Linq;
-using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Services
 {
     internal class SemanticTokensEditsDiffer : TextDiffer
     {
-        private SemanticTokensEditsDiffer(uint[] oldArray, uint[] newArray)
+        private SemanticTokensEditsDiffer(IReadOnlyList<int> oldArray, ImmutableArray<int> newArray)
         {
             if (oldArray is null)
             {
                 throw new ArgumentNullException(nameof(oldArray));
             }
 
-            if (newArray is null)
```

Comment:
Can this actually be `null` now?

Reply:
ImmutableArray<T> is a non-nullable value type, so the compiler complains if we even try to null-check.